pom.xml

```diff
@@ -802,20 +802,6 @@ under the License.
 						</executions>
 					</plugin>
 				</plugins>
-				<pluginManagement>
```

Comment:
what's the benefit of moving it out of the release profile? The profile still exists, and the plugin should only be used when making a release, no?